mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -257,7 +257,8 @@ fun JSONObject.collectWidgets(parent: Widget?): List<Widget> {
     val widget = Widget(
         getString("widgetId"),
         parent?.id,
-        optString("label", ""),
+        optString("label", "").split("[", "]")[0],
+        optString("label", "").split("[", "]").getOrNull(1),
```

Comment:
Splitting twice seems unneeded (here and above), I think using an intermediate variable would be better.

Reply:
I think it would be better to save the raw label as is in a private var and make it accessible via two functions (like it was before).

Reply:
> I think it would be better to save the raw label as is in a private var and make it accessible via two functions (like it was before).


Done